packages/ui/src/common/components/page/Sidebar/Navigation.tsx

```diff
@@ -3,15 +3,19 @@ import styled from 'styled-components'
 import { Colors, ZIndex } from '../../../constants'
 
 export const Navigation = styled.nav`
-  display: flex;
+  display: none;
   position: fixed;
   top: 0;
   left: 0;
-  grid-area: navbar;
+  grid-area: sidebar;
   width: 100%;
   max-width: 226px;
   height: 100vh;
   z-index: ${ZIndex.navbar};
+
+  @media (min-width: 1024px) {
+    display: flex;
+  }
 `
 
 export const NavigationInnerWrapper = styled.div`
```

Comment:
Finally please reduce the height of the `barheader` row to `72px` on mobile to fully match [the sliding navigation design](https://www.figma.com/file/EaObQEN95O0IWjSVTUQ5NN/Pioneer-RWD?type=design&node-id=708-202748&mode=dev)